agr_api/src/main/java/org/alliancegenome/api/controller/DiseaseController.java

```diff
@@ -474,12 +475,6 @@ public JsonResultResponse<GeneDiseaseAnnotationDocument> getDiseaseAnnotationsRi
 			}
 		}
 
-		if(filterReference != null) {
-			pagination.addFilterOption("references.crossReferences.curie", QueryParserBase.escape(filterReference) + "&PMID");
```

Comment:
The `&` logic should probably be taken out from the backend also.

Reply:
I am not sure what you mean? The filtering is done on a new field while the old filtering logic has been removed.